### ISSUE TITLE
"<title>" should be present in all pages in change-password.html

### ISSUE THREAD

**Issue by Qhuy2004** [namnguyen1409/trialproject#18]
The **"<title> should be present in all pages"** issue occurs when an HTML page is missing a <title>tag inside the<head>` section. This affects SEO, user experience, and accessibility.

Causes
The <title> tag is missing from the HTML page.
A dynamically generated page does not set an appropriate <title>.
Forgetting to add <title> in JSP files when using Servlets and JSP.

How to fix it:
Add tag < titile > in < head >
If you're doing JSP + Servlets, make sure each page has a <title>.
Create <title> dynamically in JSP
